### DISCUSSION TITLE
增加Azure/AWS/Google的TTS API接口以及语音输入功能

### DISCUSSION THREAD

**Discussion by JackyWay** [ChatGPTNextWeb/NextChat#2403] [Ideas]
如果能够增加Azure/AWS/Google的TTS（text to speech） API接口， 效果会更好，可以达到更好的交互，特别是学习语言。
还有就是翻译接口，例如可以调用谷歌等翻译。
再有就是支援Azure OpenAI的API。

**Comment by Leizhenpeng** [comment]
正在支援tts和stt

https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/pull/5329
